webapp/src/components/properties/user/user.tsx

```diff
@@ -16,24 +16,23 @@ type Props = {
 }
 
 const UserProperty = (props: Props): JSX.Element => {
-    const workspaceUsers = useContext(WorkspaceUsersContext) as Array<IUser>|undefined
-    const workspaceUsersById = useContext(WorkspaceUsersByIDContext) as Map<string, IUser>|undefined
+    const workspaceUsers = useContext(WorkspaceUsersContext) as WorkspaceUsersContextData
 
     if (props.readonly) {
-        return (<div className='UserProperty octo-propertyvalue'>{workspaceUsersById?.get(props.value)?.username || props.value}</div>)
+        return (<div className='UserProperty octo-propertyvalue'>{workspaceUsers.usersById?.get(props.value)?.username || props.value}</div>)
     }
 
     return (
         <Select
-            options={workspaceUsers}
+            options={workspaceUsers.users}
             isSearchable={true}
             isClearable={true}
             backspaceRemovesValue={true}
             className={'UserProperty'}
             styles={getSelectBaseStyle()}
             getOptionLabel={(o: IUser) => o.username}
             getOptionValue={(a: IUser) => a.id}
-            value={workspaceUsersById?.get(props.value) || null}
+            value={workspaceUsers.usersById?.get(props.value) || null}
```

Comment:
Because the type syste, I think the `?` is not needed, it should be mandatory. Can be empty, but not `null` or `undefined`.

Reply:
Done.